controllers/apply/awards-for-all/transforms.js

```diff
@@ -27,7 +27,7 @@ function transformProjectDateRange(applicationData) {
 }
 
 function transformOrgHasDifferentTradingName(applicationData) {
-    if (!!get('organisationTradingName')(applicationData)) {
+    if (get('organisationTradingName')(applicationData)) {
```

Comment:
As this is a boolean check could use `has` rather than `get`.

Reply:
Will have a look at this, empty strings might break...

Reply:
... just checked and I think this will be fine. If the field's value is `''` then it's falsey, which is what we want here.